pulp_container/app/models.py

```diff
@@ -320,26 +321,28 @@ class ContainerRemote(Remote, AutoAddObjPermsMixin, AutoDeleteObjPermsMixin):
         upstream_name (models.CharField): The name of the image at the remote.
         include_foreign_layers (models.BooleanField): Foreign layers in the remote
             are included. They are not included by default.
+        #TODO
+        include_tags
+        exclude_tags
+        sigstore
     """
 
     upstream_name = models.CharField(max_length=255, db_index=True)
     include_foreign_layers = models.BooleanField(default=False)
     include_tags = fields.ArrayField(models.CharField(max_length=255, null=True), null=True)
     exclude_tags = fields.ArrayField(models.CharField(max_length=255, null=True), null=True)
+    sigstore = models.TextField(null=True)
 
     TYPE = "container"
     ACCESS_POLICY_VIEWSET_NAME = "remotes/container/container"
 
     @property
     def download_factory(self):
         """
-        Return the DownloaderFactory which can be used to generate asyncio capable downloaders.
+        Downloader Factory that maps to custom downloaders which support registry auth.
```

Comment:
Due to flake8 nitpick: D403 First word of the first line should be properly capitalized